cmd/snap/main.go

```diff
@@ -556,7 +556,12 @@ func run() error {
 			}
 		}
 
-		msg, err := errorToCmdMessage("", strings.ToLower(parser.Active.Name), err, nil)
+		var cmdName string
+		if parser.Active != nil {
+			cmdName = parser.Active.Name
```

Comment:
I can see that we no longer strings.ToLower the cmdName, is that intentional?

Reply:
Yes, in practice it was irrelevant but IMO it wasn't consistent since we don't usually silently lowercase commands (i.e., if we run `Install` or `reFresh`, snap fails)